src/main/kotlin/de/craftan/engine/gameflow/states/AwaitStructurePlacement.kt

```diff
@@ -0,0 +1,24 @@
+package de.craftan.engine.gameflow.states
+
+import de.craftan.CraftanGameAction
+import de.craftan.engine.CraftanGame
+import de.craftan.engine.gameflow.RoundState
+import de.craftan.engine.gameflow.actions.PlaceStructureAction
+import de.craftan.engine.map.graph.StructureInfo
+
+class AwaitStructurePlacement(
```

Comment:
This is for the pregame where we wait for the player to place their FIRST strucute. Please change the name or add a javadoc to make that clear

Reply:
I think await AwaitStructurePlacement shouldn't only be for the pregame. For example when a player uses the place 2 roads card, it would be useful to also use this RoundState. It might also be possible to implement this RoundState after a player chooses to build something, but I am not so sure about that one.